tests/Feature/FileUploaderTest.php

```diff
@@ -14,18 +16,17 @@ public function a_user_can_upload_file()
             'name'      => 'test',
             'path'      => $this->getTestFilesDirectory('test.psd'),
         ];
-        $file = new UploadedFile($data['path'], $data['name'], 'image/jpeg', filesize($data['path']) , null, true);
+        $file = new UploadedFile($data['path'], $data['name'], 'image/jpeg', filesize($data['path']), null, true);
         $response = $this->call('POST', 'upload', $data, [], ['file' => $file], [], []);
 
         $response->assertStatus(200);
         $response->assertSee('psd');
     }
 
 //    /** @test */
-    public function unauthorized_user_cannot_upload_file() {
+    public function unauthorized_user_cannot_upload_file()
+    {
 //        TODO
```

Comment:
@SamuelT-ST can you please have a look on missing tests? After the PR is approved, but before merged.